judo-runtime-core-jsl-itest/src/test/resources/model/AutoMappedTransferSingleEntityModel/AutoMappedTransferSingleEntity.jsl

```diff
@@ -7,11 +7,11 @@ entity ReferenceEntity {
 }
 
 entity ContainerSingleCompositionEntity {
-    field ReferenceEntity singleCompositionEntity;
-    field required ReferenceEntity singleRequiredCompositionEntity;
+    field ReferenceEntity singleComposition;
```

Comment:
Why was it necessary to rename the field?

Reply:
For reduce the lenght of the variables